src/System.Drawing.Common/src/misc/ClientUtils.cs

```diff
@@ -172,29 +168,13 @@ private static void Debug_SequentialEnumIsDefinedCheck(System.Enum value, int mi
             if (sequentialEnumInfo == null)
             {
                 sequentialEnumInfo = new SequentialEnumInfo(t);
-
-                if (t_enumValueInfo.Count > MAXCACHE)
-                {
-                    // see comment next to MAXCACHE declaration.
-                    Debug.Fail("cache is too bloated, clearing out, we need to revisit this.");
-                    t_enumValueInfo.Clear();
```

Comment:
Yeah this whole method is in a debug only ifdef so the condition is only checked in debug mode so this will never be reached